### PR TITLE
[cmd/gokeyless] Make manual activation more robust.

### PR DESCRIPTION
**Pull request by elithrar**
- Now writes the CSR to disk
- Quits if an existing CSR (default: server.csr) is found to avoid generating a
  new key and invalidating any "in flight" CSRs.
- Requires manual removal (`rm server.csr`) to reset process.

### CODE REVIEW

**Comment by elithrar** [comment]
Note: although `-manual-activation` is a fallback, making it robust/providing a better UX is still useful.

File: cmd/gokeyless/gokeyless.go [hunk 1]
```diff
@@ -108,3 +116,24 @@ func LoadKey(in []byte) (priv crypto.Signer, err error) {
...
+	}
+
+	f, err := os.Create(csrFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	_, err = f.Write(csr)
+	if err != nil {
```
Comment (jkroll-cf) [inline cmd/gokeyless/gokeyless.go]: This could be one-lined with the `f.Write` above.

**Comment by kisom** [comment]
Aside from @jkroll-cf's comments, LGTM.

File: cmd/gokeyless/gokeyless.go (hunk 1 above)
Comment (elithrar) [inline cmd/gokeyless/gokeyless.go]: Fixed.

**Comment by elithrar** [comment]
Updated.

**Comment by jkroll-cf** [comment]
LGTM.

**Pull request merged**